server/src/main/java/io/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -243,6 +243,8 @@ public void execute()
 
   protected void validateTieredReplicants(Map<String, Integer> tieredReplicants){
     for (Map.Entry<String, Integer> entry: tieredReplicants.entrySet()) {
+      if (entry.getValue() == null)
```

Comment:
This was added but not tested. Can you add at least one test that gets to this codepath?